students/hirot_directory/session06/mailroom6_3.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python3
```

Comment:
do let us know which version you want reviewed, when there is more than one in a PR.